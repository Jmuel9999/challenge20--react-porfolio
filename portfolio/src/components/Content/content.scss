.content {
    background-color: rgb(179, 98, 52);
    width: 100%;
    height: 100px;
}