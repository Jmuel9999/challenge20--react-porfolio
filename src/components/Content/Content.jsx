import "./content.scss"

export default function Content() {
    return (
        <div className="content">
            This is the content section
        </div>
    )
}
