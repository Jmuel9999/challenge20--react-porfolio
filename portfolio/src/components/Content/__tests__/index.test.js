
import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Content from '../Content';

afterEach(cleanup);

describe('Content component', () => {
    it('renders', () => {
      render(<Content />);
    });
  
    it('matches snapshot', () => {
      const { asFragment } = render(<Content />);
  
      expect(asFragment()).toMatchSnapshot();
    });
  })
