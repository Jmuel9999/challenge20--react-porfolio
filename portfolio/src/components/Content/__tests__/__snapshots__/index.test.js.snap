// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="content"
  >
    This is the content section
  </div>
</DocumentFragment>
`;
